Documentation/error-handling.md

```diff
@@ -0,0 +1,150 @@
+## Dialing In gRPC
+
+grpc.Dial is a function in the gRPC library that creates a virtual connection from the gRPC client to the gRPC server. 
```

Comment:
It would be nicer to add backticks to all symbols. So, things like grpc.Dial, ClientConn, WithTransportCredentials() etc should be inside of backticks so that they stand out when rendered.